src/ssm.js

```diff
@@ -100,8 +103,14 @@
         fireAllMethodsInArray(leaveMethods);
         fireAllMethodsInArray(enterMethods);
         fireAllMethodsInArray(resizeMethods);
+
+        if (changed) {
```

Comment:
Changed is more legible, only change i think this pull request needs is for consistency, change stateChanged to onStateChange as all the callbacks do this, once done I will port this functionality to v3 of SSM which hopefully comes out in next couple weeks